modules/kafka/kafka.go

```diff
@@ -30,6 +30,13 @@ echo '' > /etc/confluent/docker/ensure
 /etc/confluent/docker/configure
 /etc/confluent/docker/launch`
 	// }
+
+	// starterScriptApache {
+	apacheStarterScriptContent = `#!/bin/bash
+export KAFKA_ADVERTISED_LISTENERS=%s,BROKER://%s:9092
```

Comment:
I would also add a listener for localhost and advertise it here (as well as for the case of confluentinc/confluent-local image). Feel free to check #3488. Such listener can be used by the user in custom wait strategy checking localhost (e.g. using kafka-topics tool) from command executed within container.

Reply:
This does sound interesting, but I am not quite sure if this PR should fit this part. 
This change should probably come with some kind of example that would demonstrate this technique?

Reply:
I added `TestKafkaLocalhostListener` into #3488 to demonstrate possible usage of localhost listener.

Reply:
Copied here : https://github.com/testcontainers/testcontainers-go/pull/3249/commits/b21194e6ed5b8d899f17362e7a0dc1c153c4ae78
:)
Thanks!

This is just initial, needs some kind of documentation probably to let users know how this works too :)
(and when it does not, since native images do not include those tools)

Reply:
FYI, I have also documented localhost listener usage in md file: https://github.com/testcontainers/testcontainers-go/pull/3249/commits/f6f75b2e918be850b357a5ba2fe496c26f19ec4f